### ISSUE TITLE
Train the object detection model (yolo3)

### ISSUE THREAD

**Issue by masoudrahimi39** [masoudrahimi39/ECG-code#5]
Develop a code to have the training pipeline as below steps:

it gets data (input ecg image and the bounding box as label) from data_loader/loader.py
loads the model
design the loss function
log the loss and accuracy for training and validation sets
